lib/src/main/java/com/thenewboston/api/bankapi/datasource/BankDataSource.kt

```diff
@@ -191,14 +194,15 @@ class BankDataSource @Inject constructor(private val networkClient: NetworkClien
         }
     }
 
-    suspend fun fetchInvalidBlocks(): Outcome<InvalidBlockList> = makeApiCall(
-        call = { getInvalidBlocks() },
+    suspend fun fetchInvalidBlocks(pagination: PaginationOptions = PaginationOptions()): Outcome<InvalidBlockList> = makeApiCall(
```

Comment:
No need for a default constructor. Just leave it as a parameter.

Reply:
done